specs/service_deduction.adoc

```diff
@@ -0,0 +1,174 @@
+// vim: tw=79
+
+= Service Deduction Module for Node Agent
+
+Node agents need to ship with modules that can detect various services already
+present on the nodes
+
+== Problem description
+
+This specification describes service deduction module to manage the services
+present on the nodes and populate the details; to and from the central store.
+The list could then be presented to the administrator via the API and the UI
+to allow selection of the nodes for monitoring, service operation and
+troubleshoting.
+
+== Use Cases
```

Comment:
Usecases needs to be rewritten. To me it is as simple as getting the details  intrerested services like ceph/gluster related services, centreal store related services etc, store them in etcd and present it through an API.